.github/pull_request_template.md

```diff
@@ -50,3 +52,4 @@ Finally, if you're only making changes to development scripts or tests, you may
 - [ ] I've updated the test suite for new or updated code as appropriate
 - [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
 - [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate
+- [ ] I've prepared client pull requests to resolve any breaking changes using preview builds
```

Comment:
Could it apply to dependent packages in general rather than just clients?
```suggestion
- [ ] I've prepared draft pull requests on consumer packages to resolve any breaking changes using preview builds
```

Feel free to phrase this better

Reply:
Good point, are there many usages of these core packages outside of the clients?

I just assumed that extension and mobile where the biggest consumers so wanted to highlight them directly.

Reply:
This is an example I can think about now: https://github.com/MetaMask/metamask-mobile/pull/11111#discussion_r1752031379

In this case `@metamask/phishing-controller`'s latest version is reaching the client before snaps-controller and other dependents, creating unaligned dependencies which are quite difficult to debug when it comes to messenger instances injected from the client

Reply:
Updated to mention both for clarity.